pkg/agent/deploymentController.go

```diff
@@ -39,6 +39,7 @@ func DeploymentController(kubeEnv *KubeEnv, gimletHost string, agentKey string)
 							Event:   EventDeploymentCreated,
 							Env:     kubeEnv.Name,
 							Repo:    svc.GetAnnotations()[AnnotationGitRepository],
+							Osca:    getOpenServiceCatalogAnnotations(svc),
```

Comment:
We only handle these annotations on the Kubernetes `Service` resource. And for that matter, all gimlet annotations. The `Service` identifies the app.